robosuite/controllers/config/robots/default_gr1_fixed_lower_body.json

```diff
@@ -12,9 +12,9 @@
         "ik_input_rotation_repr": "axis_angle",
         "verbose": false,
         "ik_posture_weights": {
-            "robot0_torso_waist_yaw": 10.0,
-            "robot0_torso_waist_pitch": 10.0,
-            "robot0_torso_waist_roll": 200.0,
+            "robot0_torso_waist_yaw": 30.0,
+            "robot0_torso_waist_pitch": 100.0,
+            "robot0_torso_waist_roll": 100.0,
```

Comment:
these changes are making the robot less agile. for example reaching up inside cabinets or farther back in the counter, etc. can we try to make use of the waist only when really needed? I think this part just needs more tuning, perhaps along with the overall gains of the controller.